xds/internal/balancer/edsbalancer/eds_impl.go

```diff
@@ -515,6 +515,11 @@ func (d *dropPicker) Pick(info balancer.PickInfo) (balancer.PickResult, error) {
 	}
 	if d.counter != nil {
 		if err := d.counter.StartRequest(); err != nil {
+			// Drops by circuit breaking are reported with empty category. They
+			// will be reported only in total drops, but not in per category.
+			if d.loadStore != nil {
+				d.loadStore.CallDropped("")
```

Comment:
Why do we report these with an empty category instead of using a category which makes it clear that these calls were dropped because of circuit-breaking?

Reply:
This is defined in the design doc.
And I believe the reason behind is that envoy doesn't report those with category.